### PR TITLE
Add docs for highlights, notes, archive, unarchive, and calculate ratings endpoints

### PR DESCRIPTION
**Pull request by maboelnour**
This PR adds docs for these endpoints:
- DELETE `reviews/:review_id/highlight`
- POST `reviews/:review_id/highlights`
- DELETE `reviews/:review_id/articles/:article_id/notes/:note_id`
- PUT `reviews/:review_id/articles/:article_id/notes/:note_id`
- POST `reviews/:review_id/articles/:article_id/notes`
- POST `reviews/:review_id/archive`
- POST `reviews/:review_id/unarchive`
- POST `reviews/:review_id/calculate_ratings`

### CODE REVIEW

**Review by hammady — CHANGES_REQUESTED** [review]
No text provided in the "Docs" section for each endpoint.

**Comment by maboelnour** [comment]
@hammady I added info in the `Docs` section for each request.

File: .insomnia/Request/req_26bf11ada1a445d595472cc0dfc8a61f.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 _id: req_26bf11ada1a445d595472cc0dfc8a61f
 type: Request
 parentId: fld_eeb62a531f8441478a6d2f536021400d
-modified: 1675366460007
+modified: 1675614354888
 created: 1675356164311
 url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/archive"
 name: Archive
-description: ""
+description: This endpoint archive the review
```
Comment (hammady) [inline .insomnia/Request/req_26bf11ada1a445d595472cc0dfc8a61f.yml:8]:
```suggestion
description: This endpoint archives the review
```

File: .insomnia/Request/req_fa38417da7fb46ddaa3f35859090e066.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 _id: req_fa38417da7fb46ddaa3f35859090e066
 type: Request
 parentId: fld_eeb62a531f8441478a6d2f536021400d
-modified: 1675356435313
+modified: 1675614362621
 created: 1675356227244
 url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/unarchive"
 name: Unarchive
-description: ""
+description: This endpoint unarchive the review
```
Comment (hammady) [inline .insomnia/Request/req_fa38417da7fb46ddaa3f35859090e066.yml:8]:
```suggestion
description: This endpoint unarchives the review
```

**Review by hammady — CHANGES_REQUESTED** [review]

**Review by hammady — APPROVED** [review]

**Pull request merged**